lib/geordi/db_cleaner.rb

```diff
@@ -47,9 +47,25 @@ def edit_whitelist(dbtype)
         db.sub!(@derivative_dbname, '')
         tmpfile_content.push(['drop', db])
       end
-      whitelisted_dbs.each do |db|
-        tmpfile_content.push(['keep', db])
+      warn_manual_whitelist = false
+      whitelisted_dbs.each do |db_name|
+        # Remove 'keep' word from whitelist entries. This is not normally required since geordi
+        # does not save 'keep' or 'drop' to the whitelist file on disk but rather saves a list
+        # of all whitelisted db names and just presents the keep/drop information while editing
+        # the whitelist to supply users a list of databases they can whitelist by changing the
+        # prefix to 'keep'. Everything prefixed 'drop' is not considered whitelisted and thus
+        # not written to the whitelist file on disk.
+        #
+        # However, if users manually edit their whitelist files they might use the keep/drop
+        # syntax they're familiar with.
+        if db_name.split.length > 1 && db_name.split.first == 'keep'
+          db_name = db_name.split[1..-1].join(' ')
+          warn_manual_whitelist = true
+        end
+        tmpfile_content.push(['keep', db_name]) unless db_name.empty?
       end
+      warn "Your whitelist #{whitelist} seems to have been generated manually. In that case, make sure to use only one database name per line and omit the 'keep' prefix.\n\n"\
+      'Launching the editor.' if warn_manual_whitelist
```

Comment:
Hier die gleiche Anmerkung wie in https://github.com/makandra/geordi/pull/62#discussion_r258378138.